Fixing an error in markdown parsing

mrpotatoes opened the pull request:
In reference to this bug story: https://github.com/brandonweiss/charge/issues/808

My process

```
git clone https://github.com/brandonweiss/charge.git
cd charge
yarn install
node cli.js [build|serve] docs/ tmp/
```

Receive this error for either `build` or `serve`:
```
Error: Unknown language: `text` is not registered
```

I want to mention that I'm really interested in Charge and hope it becomes bigger. I may be forking to see what I can get into it (generalize for myself and others Charge included) so I'd like to be able to play around with it more.

brandonweiss commented on lib/utilities.js:
```diff
@@ -7,7 +7,9 @@ export const importComponent = async (path) => {
     let codeFrame = error.codeFrame
-    error.stack = [description, codeFrame].join("\n")
+
+    const location = `Location: ${path}`
+    error.stack = [location, description, codeFrame].join("\n")
```
Thanks for improving this! I don’t know why I never though to log the file path here 😅 

Two things. I think “file” is probably a slightly clearer name. And I think we can lop off the beginning of the path and make it relative. So instead of `File: /Users/brandon/Code/charge/docs/pages.html.mdx` it would be `File: docs/pages.html.mdx`. That should make the file path stand out a bit more.

```suggestion
    const relativePath = path.replace(`${process.cwd()}${separator}`, "")
    const file = `File: ${relativePath}`
    error.stack = [file, description, codeFrame].join("\n")
```

This “path” import at the top would also need to be changed but I can’t seem to add a comment there to make the patch suggestion.

```diff
- import { normalize } from "path"
+ import { normalize, sep as separator } from "path"
```

mrpotatoes commented:
Fixed it and added  newline so that it it doesn't get lost in the console. I would suggest adding `colors` so that it's red and then you can't miss it if you tried. Maybe even 

```
[builder] › ⚙️  starting  Building static files from docs

File: docs/index.html.mdx
Error: Unknown language: `txt` is not registered

error Command failed with exit code 1.
```
